🔧(project) improve production documentation (#16)
apui-avignon-university/warren-tdbp: production/README.md

docker ps
```

> If a service is not started, we can investigate further with:

Reviewer: ```suggestion
If a service has not been started, we can investigate further with:

```

```bash
docker service ps --no-trunc SERVICE_NAME
```

For investigating further in case a service is not healthy, use:
```bash
docker inspect --format "{{json .State.Health }}" CONTAINER_ID | jq
```
_Note_: this command requires that [jq](https://jqlang.github.io/jq/) is installed
on your operating system.

> We can check logs of a container directly with the command:

Reviewer: ```suggestion
We can check logs of a specific container directly with the command:
```

### Configuring Moodle web services for XI indexation 

In order for the Experience Index to access the Moodle courses and their content, we

> need to configure a web service under `Moodle > Web services` (or `Administration > Plugins > Web service`) and following the 10 steps

Reviewer: ```suggestion
need to configure a web service under `Moodle > Web services` 
(or `Administration > Plugins > Web service`) and following the 10 steps
```
